docs/Theming.md

```diff
@@ -284,14 +284,16 @@ const App = () => (
 );
 ```
 
+Note that we use `createMuiTheme` from `react-admin`. This ensures your theme contains the custom values needed for some `react-admin` components such as the `<SideBar>`.
```

Comment:
I suggest that we don't make that a requirement (or it would be another BC break) and instead provide default values for these theme dimensions in the sidebar component.